packages/astro/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@spotlightjs/astro",
   "description": "Add spotlight to your Astro toolbar, for better debugging.",
-  "version": "1.0.19",
+  "version": "2.0.0-alpha.0",
```

Comment:
Usually, this step would happen in the "Prepare Publish" workflow but changesets handles pre-releases a bit differently. In the hope of not having to do too many, for now I opted to just carry out the chagneset pre-release preparations manually and I'll publish directly from `v2-dev` once this is merged. Should work 🤞